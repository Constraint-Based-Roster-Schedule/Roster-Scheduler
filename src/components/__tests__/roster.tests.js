import { render, screen } from "@testing-library/react";
import { getByTestId } from "@testing-library/react";
import IndividualRoster from "../individualRoster";
import ReactDom from "react-dom";
import { Button } from "react-bootstrap";
import renderer from "react-test-renderer";
import { DoDisturb } from "@mui/icons-material";
// it("renders without crashing testing", () => {
//   const div = document.createElement("div");
//   ReactDom.render(<Button></Button>, div);
// });

test("roster page testing", () => {
  render(<IndividualRoster />);
  
});
it('matches snapshot',()=>{
  const tree=renderer.create(<IndividualRoster></IndividualRoster>).toJSON();
  expect(tree).toMatchSnapshot();

})