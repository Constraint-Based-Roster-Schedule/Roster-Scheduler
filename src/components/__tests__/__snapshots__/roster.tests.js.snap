// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  data-testid="individual-roster"
>
  <div
    className="loader-container"
    data-testid="loader"
  >
    <div
      className="sk-fade-in sk-spinner spinner ball-spin-fade-loader"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
